backend_site/views.py

```diff
@@ -0,0 +1,37 @@
+"""Basic routes for static resources"""
+
+from flask import Blueprint
+from .settings import STATIC_DIR, STATIC_ROUTE
+
+blueprint = Blueprint('static', __name__,
+                      static_folder=STATIC_DIR,
+                      static_url_path=STATIC_ROUTE)
+
+# No CORS on static routes!
+
+print("Static blueprint root path:")
+print(blueprint.root_path)
+print("Static blueprint static directory:")
+print(blueprint.static_folder)
+
+
+@blueprint.route('')
+def homePage():
+    """Route for '/' - Home Page"""
+    # print("Firing homepage route!")
```

Comment:
Don't forget to remove old lines of code from making this PR if they aren't necessary

Reply:
removed